scaladoc/test-source-links/dotty/tools/scaladoc/source-links/RemoteLinksTest.scala

```diff
@@ -38,16 +38,17 @@ class RemoteLinksTest:
   def runTest =
     assertTrue(mtslAll.nonEmpty)
     val mtsl = randomGenerator.shuffle(mtslAll).take(80) // take 80 random entries
-    val pageToMtsl: Map[String, List[(String, String)]] = mtsl.groupMap(_._2.split("#L").head)(v => (v._1, v._2.split("#L").last))
-    pageToMtsl.foreach { case (link, members) =>
+    val pageToMtsl: Map[String, List[(String, String)]] =
```

Comment:
```suggestion
    val pageToMtsl: Map[String, List[(String, Int)]] =
```

... if I eyeball correctly